homeassistant/components/google_cloud/tts.py

```diff
@@ -24,7 +24,7 @@
 SUPPORTED_LANGUAGES = [
     'da-DK', 'de-DE', 'en-AU', 'en-GB', 'en-US', 'es-ES', 'fr-CA', 'fr-FR',
     'it-IT', 'ja-JP', 'ko-KR', 'nb-NO', 'nl-NL', 'pl-PL', 'pt-BR', 'pt-PT',
-    'ru-RU', 'sk-SK', 'sv-SE', 'tr-TR', 'uk-UA',
+    'ru-RU', 'sk-SK', 'sv-SE', 'tr-TR', 'uk-UA', 'hu-HU'
```

Comment:
Please add it in the correct alphabetical order (between fr-FR and it-IT)

Reply:
OK!